File: a03/Main.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int N = sc.nextInt();
        int K = sc.nextInt();
        int[] P = new int[N];
        int[] Q = new int[N];
        for (int i = 0; i < N; i++) {
            P[i] = sc.nextInt();
        }
        for (int i = 0; i < N; i++) {
            Q[i] = sc.nextInt();
        }
        boolean Answer = false;
        for (int x = 0; x < N; x++) {
            for (int y = 0; y < N; y++) {
                if (P[x] + Q[y] == K) {
                    Answer = true;
                }
            }
        }
        if (Answer) {
            System.out.println("Yes");
        } else {
            System.out.println("No");
        }
    }
}

File: a03/main.php
<?php
fscanf(STDIN, "%d %d", $N, $K);
$P = array_map('intval', explode(' ', trim(fgets(STDIN))));
$Q = array_map('intval', explode(' ', trim(fgets(STDIN))));
$Answer = false;
for ($x = 0; $x < $N; $x++) {
    for ($y = 0; $y < $N; $y++) {
        if ($P[$x] + $Q[$y] == $K) {
            $Answer = true;
        }
    }
}
if ($Answer) {
    echo "Yes\n";
} else {
    echo "No\n";
}
?>
